juju/juju
# Drop support for unmanaged Azure storage

Azure unmanaged storage has been deprecated for a long time. We stopped supporting it before Juju 2.3 was shipped. Azure will drop support for it in 2023.
This PR drops the legacy support in Juju - only managed disks are supported.

Any really, really, really old models [will need to be migrated](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/migrate-to-managed-disks) out of band.

## QA steps

juju bootstrap azure
juju deploy postgresql --storage pgdata=1G,azure
juju destroy-model

Use the Azure portal to inspect that things are created and deleted as expected.
